ReactiveCocoa/Swift/Signal.swift

```diff
@@ -273,26 +273,29 @@ public func combineLatestWith<T, U, E>(otherSignal: Signal<U, E>)(signal: Signal
 	return Signal { observer in
 		let lock = NSRecursiveLock()
 		lock.name = "org.reactivecocoa.ReactiveCocoa.combineLatestWith"
-
+		
```

Comment:
Can you please fix the whitespace changes in this file, so lines unrelated to the diff don't change?